my-app/src/App.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import { people } from './people.js';
+import PeopleList from './PeopleList'
+import './App.css';
+
+class App extends Component {
+  constructor(){
+    super();
+    this.state = {
+      People: []
+    };
+  }
+
+  componentDidMount(){
+    this.setState({ People: people });
+   }
+
+  render() {
+    return (
+      <div className="App">
+        <header className="App-header">
+          <img src={logo} className="App-logo" alt="logo" />
+          <h1 className="App-title">Welcome to React</h1>
+        </header>
+        <p className="App-intro">
+          <PeopleList people = {this.state.People}/>
```

Comment:
There's an error `Warning: validateDOMNesting(...): <div> cannot appear as a descendant of <p>.` in your code that stems from this, you have a `<p>` element that is currently housing `PeopleList` as a descendant (which is a div) and so it's complaining. I think you can just make this <p> a <div> and things would be okay.